src/Raven.Server/Documents/DocumentDatabase.cs

```
@@ -168,6 +171,20 @@ public DocumentDatabase(string name, RavenConfiguration configuration, ServerSto
                 });
                 _hasClusterTransaction = new AsyncManualResetEvent(DatabaseShutdown);
                 IdentityPartsSeparator = '/';
+
+                this.IndexWritesSynchronization = new StorageEnvironmentSynchronization();
```

Move this to the `case`'s `default`?

---

Yes, I am a bit hesitant if `switch` `default` because they tend to mask errors, but in this case which is essentially an ignore, there is no reason why not.